backend/util/tracer-graphql.go

```diff
@@ -32,18 +33,18 @@ func (t Tracer) Validate(graphql.ExecutableSchema) error {
 }
 
 func (t Tracer) InterceptField(ctx context.Context, next graphql.Resolver) (interface{}, error) {
-	start := time.Now()
 	// taken from: https://docs.datadoghq.com/tracing/setup_overview/custom_instrumentation/go/#manually-creating-a-new-span
 	fc := graphql.GetFieldContext(ctx)
-	fieldSpan, ctx := StartSpanFromContext(ctx, "operation.field", ResourceName(fc.Field.Name))
-	fieldSpan.SetAttribute("field.type", fc.Field.Definition.Type.String())
+	fieldSpan, ctx := tracer.StartSpanFromContext(ctx, "operation.field", tracer.ResourceName(fc.Field.Name))
```

Comment:
switch to only datadog tracing in this highlight-only util file. eventually, this file can be dropped. highlight gqlgen tracing happens via the `sdk/highlight-go/tracer.go` file